main.go

```diff
@@ -272,6 +288,8 @@ func main() {
 
 	cmdErr := command.Run()
 
+	fmt.Println()
+	logExitStatus(timeoutcmd.ExitStatus(cmdErr))
```

Comment:
NOTE: you could avoid importing `timeoutcmd` by calling `RunAndReturnExitCode` on the command model.